problems/strings.js

```diff
@@ -3,23 +3,29 @@
  * @param {string} - str
  * @returns {string} - first char
  */
-function firstChar() {}
+const firstChar = (str) => {
+  return str.slice(0, 1);
+};
 
 /**
  * Returns the length of a string
  * @param {string} - str
  * @returns {number} - length of string
  */
 
-function stringLength() {}
+const stringLength = (str) => {
+  return str.length;
+};
 
 /**
  * Returns the last character of any string
  * @param {string} - str
  * @returns {string} - last letter
  */
 
-function lastLetter() {}
+const lastLetter = (str) => {
+  return str.slice(str.length - 1);
```

Comment:
There's also an easier way to do this. Think of how we find the last element of an array